common/persistence/sql/sqlplugin/postgresql/db.go

```diff
@@ -32,7 +32,7 @@ import (
 	"go.temporal.io/server/common/persistence/schema"
 	"go.temporal.io/server/common/persistence/sql/sqlplugin"
 	"go.temporal.io/server/common/persistence/sql/sqlplugin/postgresql/driver"
-	postgresqlschemaV96 "go.temporal.io/server/schema/postgresql/v96"
+	postgresqlschemaV12 "go.temporal.io/server/schema/postgresql/v12"
```

Comment:
what is the reason we can't import it as postgresqlschema?

Reply:
I was just keeping the pattern. I don't mind either way.